mgmt/rest/v2/mock/mock_task_manager.go

```diff
@@ -179,7 +179,7 @@ const (
       "deadline": "4ns",
       "creation_timestamp": -62135596800,
       "last_run_timestamp": -1,
-      "task_state": "Running",
+      "state": "Running",
```

Comment:
We cannot change the response of the API after it's landed (V1 or V2).  Please revert this back to `task_state`.